src/data_collection.py

```diff
@@ -253,6 +286,10 @@ def print_individual_in_table(file_name):
         "Modified Lines",
         "Lines/Commit",
         "File Types",
+        "Commits to Testing",
+        "Commits Elsewhere",
+        "Percent of Commits to Testing",
+        "Percent of Commits not to Testing",
```

Comment:
In the latest changes to the master branch, we significantly changed the way the table is printed. The function now accepts either a dictionary or a `.json` file which needs to be specified in the function call. The headings for columns are also now sent in as a list of keys for the dictionary. Please merge master into your branch to get those changes.